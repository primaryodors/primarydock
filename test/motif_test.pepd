
LET $prot = "pdbs/OR1/OR1A1.upright.pdb"
IF $arg2 THEN LET $prot = $arg2

LOAD $prot
ECHO $SEQUENCE
ECHO "Searching for motifs..."

SEARCH 1 %SEQLEN "HTPMYFFL" %motif
IF %motif > 0 THEN ECHO "Found HTPMYFFL motif at " %motif

SEARCH 50 %SEQLEN "PKML" %motif
IF %motif > 0 THEN ECHO "Found PKML motif at " %motif

SEARCH 1 %SEQLEN "MAYDRYVAICXPLXY" %motif
IF %motif > 0 THEN ECHO "Found MAYDRYVAICXPLXY motif at " %motif

SEARCH 100 %SEQLEN "HFFCD" %motif
IF %motif > 0 THEN ECHO "Found HFFCD motif at " %motif

SEARCH 1 %SEQLEN "GRXKAFSTCASHLXVV" %motif
IF %motif > 0 THEN ECHO "Found GRXKAFSTCASHLXVV motif at " %motif

SEARCH 1 %SEQLEN "PMLNPLIYSLRNKD" %motif
IF %motif > 0 THEN ECHO "Found PMLNPLIYSLRNKD motif at " %motif

