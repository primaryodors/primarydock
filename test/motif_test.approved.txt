309 MRENNQSSTLEFILLGVTGQQEQEDFFYILFLFIYPITLIGNLLIVLAICSDVRLHNPMYFLLANLSLVDIFFSSVTIPKMLANHLLGSKSISFGGCLTQMYFMIALGNTDSYILAAMAYDRAVAISRPLHYTTIMSPRSCIWLIAGSWVIGNANALPHTLLTASLSFCGNQEVANFYCDITPLLKLSCSDIHFHVKMMYLGVGIFSVPLLCIIVSYIRVFSTVFQVPSTKGVLKAFSTCGSHLTVVSLYYGTVMGTYFRPLTNYSLKDAVITVMYTAVTPMLNPFIYSLRNRDMKAALRKLFNKRISS
Searching for motifs...
Found MRENNQSSTL motif at 188
Found HTPMYFFL motif at 139
Found PKML motif at 196
Found MAYDRYVAICXPLXY motif at 118
Found HFFCD motif at 176
Found GRXKAFSTCASHLXVV motif at 232
Found PMLNPLIYSLRNKD motif at 281
Found terminal motif at 293
